packages/ui-components/stencil.config.ts

```diff
@@ -51,6 +51,7 @@ export const config: Config = {
 		}
 	],
 	plugins: [sass()],
+	sourceMap: true,
```

Comment:
we should only generate sourceMaps for development purposes.

Reply:
Do you think [this](https://github.com/kelvininc/ui-components/pull/204/commits/507a9bda0e77df01388ba32a0f68443e61087fa6) is a good solution?

Reply:
Hmmm, is it possible to use the environment variables to generate the source maps? We can have 2 commands as you do but with the same configuration (conditional to the environment).

Reply:
I tried to find a way to use the `--dev` for generating the source maps but I didn't find a way in the documentation to create conditional configuration target. Do you know how?